querydsl-examples/querydsl-example-ksp-codegen/build.gradle.kts

```diff
@@ -6,25 +6,26 @@ val querydslVersion = findProperty("querydsl.version") as String
 val assertjVersion = findProperty("assertj.version") as String
 
 plugins {
-	kotlin("jvm")
-	id("com.google.devtools.ksp")
-	kotlin("plugin.jpa")
-	kotlin("plugin.serialization")
+        kotlin("jvm")
```

Comment:
@Gyeom @IceBlizz6 

Is there any tool we can add to the build to guarantee consistent formatting?

Maven poms have sortpom, java sources google formatter.... gradle and kotlin have the last editor

Reply:
Also coverage test reporting if you guys know how

Reply:
@velo

I’ll review tools like Spotless that can be applied to this project to ensure consistent formatting across the codebase.